packages/openapi-framework/index.ts

```diff
@@ -76,11 +76,21 @@ export default class OpenAPIFramework implements IOpenAPIFramework {
   private securityHandlers;
   private validateApiDoc;
   private validator;
+  private logger;
 
   constructor(protected args = {} as OpenAPIFrameworkConstructorArgs) {
     this.name = args.name;
     this.featureType = args.featureType;
     this.loggingPrefix = args.name ? `${this.name}: ` : '';
+    this.logger = args.logger ? args.logger : console;
+    // monkey patch for node v6:
```

Comment:
perhaps we should just remove v6 support?  it's going to be EOL in 3 months anyway.

Reply:
i'd def vote for that :) it's not like this can't be used on node v6, they just have to wrap console and provide .debug()